plugins/outputs/application_insights/README.md

```diff
@@ -0,0 +1,13 @@
+# Application Insights Output Plugin
+
+This plugin writes telegraf metrics to Auzre Application Insights
+
+### Configuration
```

Comment:
I guess I don't know a whole lot about this system, but there may be some value in specifying things like RoleName, RoleInstance, and custom properties -- all things that would just end up in the TelemetryClient.Context() object.  These help isolate regions, resource groups, and instances in our queries.  Perhaps the default (hostname in RoleInstance) is enough, but I find myself always filling in these fields for services that I monitor.

Reply:
@jjjordanmsft This is good point and we are having some discussions about how to set "special" properties on AppInsights telemetry. I will add you to the email thread